### DISCUSSION TITLE
Entering new tasks into a filtered view

### DISCUSSION THREAD

**Discussion by Mikethekidmyers** [community/community#7366] [Projects and Issues] [answered]
It would be great if new tasks entered into a filtered view would by default have the attributes necessary to appear in that specific view. For example I have views filtered by milestones:

![image](https://user-images.githubusercontent.com/36883851/140526757-0e80538d-ba53-45e9-857a-5a86fc11d777.png)

And while adding new items to this view they disappear as they are added and I need to have no filters to view them before assigning them to the view they are supposed to be in.

**Comment by azenMatt** [comment, marked as answer]
We just released this change - https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/ 🎉 

Where possible, we will now add the value. Check it out and let me know if it helps!

**Reply by HonzaMikula** [reply]
We need this for Drafts as well.
